README.md

```diff
@@ -55,11 +56,33 @@ Here's a collection of blog posts (in chronological order) related to `jsii`:
 > :information_source: If you wrote blog posts about `jsii` and would like to have them referenced here, do not hesitate
 > to file a pull request to add the links here!
 
-# :gear: Contributing
+## :gear: Maintenance & Support
```

Comment:
Pulling the version support table above contributors, because this is important for people to see.